README.md

```diff
@@ -7,65 +7,65 @@ Pico
 [![Freenode IRC Webchat](https://picocms.github.io/badges/pico-chat.svg)](https://webchat.freenode.net/?channels=%23picocms)
 [![Open Bounties on Bountysource](https://www.bountysource.com/badge/team?team_id=198139&style=bounties_received)](https://www.bountysource.com/teams/picocms)
 
-Pico is a stupidly simple, blazing fast, flat file CMS.
+Pico è un CMS stupidamente semplice, incredibilmente veloce, basato su files di testo.
```

Comment:
You should change `files` to `file`, in italian you don't make plurals of foreign words.

Reply:
Thanks!